scripts/demo/demo_benchmark_app.bat

```diff
@@ -145,73 +145,75 @@ echo.
 CALL :delay 3
 
 if "%PROCESSOR_ARCHITECTURE%" == "AMD64" (
-   set "PLATFORM=x64"
+    set "PLATFORM=x64"
 ) else (
-   set "PLATFORM=Win32"
+    set "PLATFORM=Win32"
 )
 
 set VSWHERE="false"
 if exist "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" (
-   set VSWHERE="true"
-   cd "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer"
+    set VSWHERE="true"
+    cd /d "%ProgramFiles(x86)%\Microsoft Visual Studio\Installer"
 ) else if exist "%ProgramFiles%\Microsoft Visual Studio\Installer\vswhere.exe" (
-      set VSWHERE="true"
-      cd "%ProgramFiles%\Microsoft Visual Studio\Installer"
+    set VSWHERE="true"
+    cd /d "%ProgramFiles%\Microsoft Visual Studio\Installer"
 ) else (
-   echo "vswhere tool is not found"
+    echo "vswhere tool is not found"
 )
 
-set MSBUILD_BIN=
-set VS_PATH=
-
 if !VSWHERE! == "true" (
-   for /f "usebackq tokens=*" %%i in (`vswhere -latest -products * -requires Microsoft.Component.MSBuild -property installationPath`) do (
-      set VS_PATH=%%i
-   )
-   if exist "!VS_PATH!\MSBuild\14.0\Bin\MSBuild.exe" (
-      set "MSBUILD_BIN=!VS_PATH!\MSBuild\14.0\Bin\MSBuild.exe"
-   )
-   if exist "!VS_PATH!\MSBuild\15.0\Bin\MSBuild.exe" (
-      set "MSBUILD_BIN=!VS_PATH!\MSBuild\15.0\Bin\MSBuild.exe"
-   )
-   if exist "!VS_PATH!\MSBuild\Current\Bin\MSBuild.exe" (
-      set "MSBUILD_BIN=!VS_PATH!\MSBuild\Current\Bin\MSBuild.exe"
-   )
+    for /f "usebackq tokens=*" %%i in (`vswhere -latest -products * -requires Microsoft.Component.MSBuild -property installationPath`) do (
+       set VS_PATH=%%i
```

Comment:
This is underindented still; it's indented by 7 spaces instead of 8.

Reply:
Fixed, thanks.